docs/download.md

```diff
@@ -86,78 +89,19 @@ versions of the JVM with the JDBC specification implemented.
 	<th>JDBC 4.2</th>
 	<th>Source</th>
 </tr>
-<tr>
-	<td>42.1.1</td>
-	<td><a href="download/postgresql-42.1.1.jre6.jar">42.1.1 JDBC 4</a></td>
-	<td><a href="download/postgresql-42.1.1.jre7.jar">42.1.1 JDBC 41</a></td>
-	<td><a href="download/postgresql-42.1.1.jar">42.1.1 JDBC 42</a></td>
-	<td><a href="download/postgresql-jdbc-42.1.1.src.tar.gz">42.1.1 JDBC Source</a></td>
-</tr>
-
-<tr>
-	<td>42.1.0</td>
-	<td>n/a</td>
-	<td><a href="download/postgresql-42.1.0.jre7.jar">42.1.0 JDBC 41</a></td>
-	<td><a href="download/postgresql-42.1.0.jar">42.1.0 JDBC 42</a></td>
-	<td><a href="download/postgresql-jdbc-42.1.0.src.tar.gz">42.1.0 JDBC Source</a></td>
-</tr>
-
-<tr>
-	<td>42.0.0</td>
-	<td><a href="download/postgresql-42.0.0.jre6.jar">42.0.0 JDBC 4</a></td>
-	<td><a href="download/postgresql-42.0.0.jre7.jar">42.0.0 JDBC 41</a></td>
-	<td><a href="download/postgresql-42.0.0.jar">42.0.0 JDBC 42</a></td>
-	<td><a href="download/postgresql-jdbc-42.0.0.src.tar.gz">42.0.0 JDBC Source</a></td>
-</tr>
-
-<tr>
-	<td>9.4.1212</td>
-	<td><a href="download/postgresql-9.4.1212.jre6.jar">9.4.1212 JDBC 4</a></td>
-	<td><a href="download/postgresql-9.4.1212.jre7.jar">9.4.1212 JDBC 41</a></td>
-	<td><a href="download/postgresql-9.4.1212.jar">9.4.1212 JDBC 42</a></td>
-	<td><a href="download/postgresql-jdbc-9.4.1212.src.tar.gz">9.4.1212 JDBC Source</a></td>
-</tr>
-
-<tr>
-	<td>9.4.1211</td>
-	<td><a href="download/postgresql-9.4.1211.jre6.jar">9.4.1211 JDBC 4</a></td>
-	<td><a href="download/postgresql-9.4.1211.jre7.jar">9.4.1211 JDBC 41</a></td>
-	<td><a href="download/postgresql-9.4.1211.jar">9.4.1211 JDBC 42</a></td>
-	<td><a href="download/postgresql-jdbc-9.4.1211.src.tar.gz">9.4.1211 JDBC Source</a></td>
-</tr>
-
-<tr>
-	<td>9.4.1210</td>
-	<td><a href="download/postgresql-9.4.1210.jre6.jar">9.4.1210 JDBC 4</a></td>
-	<td><a href="download/postgresql-9.4.1210.jre7.jar">9.4.1210 JDBC 41</a></td>
-	<td><a href="download/postgresql-9.4.1210.jar">9.4.1210 JDBC 42</a></td>
-	<td><a href="download/postgresql-jdbc-9.4.1210.src.tar.gz">9.4.1210 JDBC Source</a></td>
-</tr>
-
-<tr>
-	<td>9.4.1209</td>
-	<td><a href="download/postgresql-9.4.1209.jre6.jar">9.4.1209 JDBC 4</a></td>
-	<td><a href="download/postgresql-9.4.1209.jre7.jar">9.4.1209 JDBC 41</a></td>
-	<td><a href="download/postgresql-9.4.1209.jar">9.4.1209 JDBC 42</a></td>
-	<td><a href="download/postgresql-jdbc-9.4.1209.src.tar.gz">9.4.1209 JDBC Source</a></td>
-</tr>
 
+{% for post in site.categories.new_release reversed %}
```

Comment:
I've restored 9.4.1212, 9.4.1211, etc versions in download.md
Only 42.x.y are converted to `_posts/...` for now.